src/solutions/components/SolutionsPanel.tsx

```diff
@@ -0,0 +1,177 @@
+import {
+  Box,
+  Collapse,
+  Heading,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  useBreakpointValue,
+} from "@chakra-ui/react"
+import { useAddRewardContext } from "components/[guild]/AddRewardContext"
+import useGuild from "components/[guild]/hooks/useGuild"
+import { usePostHogContext } from "components/_app/PostHogProvider"
+import ControlledSelect from "components/common/ControlledSelect"
+import SegmentedControl from "components/common/SegmentedControl"
+import SearchBar from "components/explorer/SearchBar"
+import { AnimatePresence } from "framer-motion"
+import { useState } from "react"
+import { FormProvider, useController, useForm } from "react-hook-form"
+import rewards from "rewards"
+import {
+  SolutionName,
+  categories,
+  engagement,
+  memberships,
+  solutions,
+  sybil,
+  tokens,
+} from "solutions"
+import { PlatformName, PlatformType } from "types"
+import Category from "./Category"
+
+const SolutionsPanel = ({
+  addReward,
+  setAddPanel,
```

Comment:
hm, I think we somehow would need to separate solutions from the add reward context, as solutions are not exactly the same as rewards (they do have an overlap though)
maybe a completely separate context? or do you think it would be a bit of an overkill?

Reply:
I actually like the idea which you implemented now! I think that a completely separate context would actually cause more confusion than the current solution.